t/ingredients.t

```diff
@@ -354,15 +354,6 @@ my @tests = (
 		}
 	],
 
-	# from vegetal origin
```

Comment:
Removed by error?

Reply:
I can not see when it was deleted in the history. I saw that you made the change. Maybe I forgot to pull the newest version and did not see any warning.